php-checks/src/test/resources/checks/HardCodedCredentialsCheck.php

```diff
@@ -67,3 +67,22 @@ function foo() {
 
 // The literal string doesn't contain the wordlist item matched on the variable name
 const DEFAULT_AMQP_PASSWORD = 'pwd'; // Noncompliant
+
+ldap_bind("a", "b", "password"); // Noncompliant
```

Comment:
I don't think that `"password"` is a good value here as it's a value which we look for in string literals. Suggestion: "secret" or "p4ssw0rd" or something else...